export class CauseNotFoundError extends Error {
  constructor() {
    super("Causa n√£o encontrada")
  }
}